my-dsl.md

```diff
@@ -1,38 +1,68 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+_What is the name of the language? Link the name to its webpage
+ (if appropriate)._
+
+The launchd system. 
 
 
 # Domain
 _Describe the language's domain in five words._
 
+Managing processes/scripts for OSX.
+
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+On startup, launchd reads property lists files in certain directories. Then
+depending on the property list (`plist`), the process is run at the specified
+intervals or otherwise stopped or managed.
+
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+launchd is "purely" domain-specifice. A launchd property list has less than
+20 possible properties. I don't think you can set variables in these `plist`s.
+In terms of use, people only use it to manage processes and scripts written
+in other languages.
+
```

Comment:
That's a really good point. On one hand, launchd's `plist`s are really easy to read, because you'd only need to read the dictionary with descriptive key names. But on the other hand, the files are pretty tedious to write. There's a lot of other XML setup required, setup that isn't necessarily coder friendly; this definitely hurts it's DSL-ness.